addons/account/models/account_move.py

```diff
@@ -376,6 +376,7 @@ def _sequence_fixed_regex(self):
     company_currency_id = fields.Many2one(
         string='Company Currency',
         related='company_id.currency_id', readonly=True,
+        store=True, precompute=True,
```

Comment:
I have no idea why the CI is not yelling, but changing the database schema is not possible on a stable version anyways